### DISCUSSION TITLE
Don't restart Julia kernel from chapter to chapter in Quarto Book

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [quarto-dev/quarto-cli#10230] [Q&A]
### Description

I am having a Quarto book with fairly small chapters and linear content that incrementally builds on previous chapters.

Unfortunately, it seems like rendering opens fresh Julia kernels for each chapter, which makes it impossible to re-use code and results from previous chapters. Is there a way to make Quarto re-use the same kernel between different files?

Somewhat related to https://github.com/quarto-dev/quarto-cli/discussions/4201

**Comment by cscheid** [comment]
This is by design, and it happens in engines other than Jupyter. Chapters need to behave independently of each other to allow incremental previewing, and so it's not a feature we will add.

You can use something like the Jupyter cache (or you can roll your own as well.)
